uiowa/uiowa
# [Bug] People list block website field

Resolves https://github.com/uiowa/uiowa/issues/7203
Resolves https://github.com/uiowa/uiowa/issues/5905

# How to test
```
ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /debug-one-people-list-block
```
- Code review
- Test https://github.com/uiowa/uiowa/issues/7203#issuecomment-1852702948 and confirm that no update hook is needed
- Do we need to do anything about https://github.com/uiowa/uiowa/issues/7203#issuecomment-1852721124?
- Edit the block and uncheck the website field and confirm that the website field is visible on save. 
- Confirm website is showing on https://sandbox.prod.drupal.uiowa.edu/leadership-team and https://sandbox.prod.drupal.uiowa.edu/featured-content-0
